scripts/azureml-assets/azureml/assets/release/model_publish_utils/mlflow_model_utils.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import logging
+import os
+import azureml.evaluate.mlflow as mlflow
```

Comment:
>mlflow [](http://example.com/codeflow?start=34&length=6)

(nit) rename to azureml_mlflow to avoid conflation with native mlflow.

Reply:
hf_mlflow